fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -25,179 +40,112 @@ NONE_METHOD_OPT="NONE"
 # --------------------------------------------------------------------------------
 #  Usage Description Function
 # --------------------------------------------------------------------------------
+opts_SetScriptDescription "Run fMRIVolume processing"
 
-script_name=$(basename "${0}")
-
-show_usage() {
-	cat <<EOF
+opts_AddMandatory '--studyfolder' 'Path' 'path' "folder containing all subject" "--path"
 
-${script_name}: Run fMRIVolume processing pipeline
+opts_AddMandatory '--subject' 'Subject' 'subject ID' ""
 
-Usage: ${script_name} [options]
+opts_AddMandatory '--fmritcs' 'fMRITimeSeries' 'file' 'input fMRI time series (NIFTI)'
 
-  [--help] : show usage information and exit
-  --path=<path to study folder>
-  --subject=<subject ID>
-  --fmritcs=<input fMRI time series (NIFTI)>
-  --fmriname=<name (prefix) to use for the output>
-  --fmrires=<final resolution (mm) of the output data>
+opts_AddMandatory '--fmriname' 'NameOffMRI' 'string' 'name (prefix) to use for the output'
 
-  --biascorrection=<method to use for receive coil bias field correction>
+opts_AddMandatory '--fmrires' 'FinalfMRIResolution' 'number' 'final resolution (mm) of the output data'
 
-        "SEBASED"
-             use bias field derived from spin echo images, must also use --dcmethod="${SPIN_ECHO_METHOD_OPT}"
+opts_AddMandatory '--biascorrection' 'BiasCorrection' 'SEBASED OR LEGACY OR NONE' "SEBASED: use bias field derived from spin echo images, must also use --dcmethod='${SPIN_ECHO_METHOD_OPT}'
```

Comment:
Formatting: the first line gets put after the `-` in the line containing the option switch, please make it look more like the original, with the first line being something like "method for receive bias correction, accepted values are:"

Reply:
I think formatting is now fixed please let me know if it still does not match, not sure how to do indentations or tabs with the new arg library.

Reply:
Don't use tabs, ever, they aren't dealt with because they have a strange width per character.  Don't do indentations without a good reason (and when they are needed, use only spaces).  We just want it to look decent and read smoothly.  Check what it looks like by running the script without arguments, and change as needed.